LibreNMS/Alert/Transport/Matrix.php

```diff
@@ -44,7 +44,8 @@ public function deliverAlert(array $alert_data): bool
 
         $message = SimpleTemplate::parse($message, $alert_data);
 
-        $body = ['body' => strip_tags($message), 'formatted_body' => "$message", 'msgtype' => 'm.text', 'format' => 'org.matrix.custom.html'];
+        $msgtype = $alert_data['severity'] == 'critical' ? 'm.notice' : 'm.text';
```

Comment:
it should always be notice, its not up to us to decide what you get alerted for and not

Reply:
Should revert my commit then.